sopel/modules/shrug.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+import sopel.module
+
+
+@sopel.module.commands('shrug')
```

Comment:
If you tweak the import above as suggested, this can just be `@commands('shrug')`.